cpp/ql/src/semmle/code/cpp/exprs/Expr.qll

```diff
@@ -250,32 +252,32 @@ class Expr extends StmtParent, @expr {
   }
 
   /**
-   * Holds if this expression has undergone an lvalue-to-rvalue conversion to
+   * Holds if this expression has undergone an *lvalue*-to-**rvalue** conversion to
```

Comment:
What's the logic for using * (italic?) and ** (bold???) widely in these commits, rather than backticks?

(previously we've always used backticks when quoting both QL code / names and C/C++ code in QLDoc, and nothing when using plain English words that describe language features)

Reply:
I use _italics_ for emphasis, and also for "foreign" words such as _lvalue_, _xvalue_, etc.  Is this also problematic?